_dashboards/management/multi-data-sources.md

```diff
@@ -128,89 +135,212 @@ data_source.authTypes:
+3. Choose a channel from the list to view or manage its settings. 
+  - Edit the channel's settings by selececting the **Actions** button and choosing the **Edit** option. Enter the required information in the **Edit channel** panel and then choose **Save**.
+  - Send a test message to the channel by selecting the **Send test message** button on the **Edit channel** window. Alternatively, you can select the **Actions** button on the channel details window and then choose the **Send test message** option from the dropdown.
+
+The following GIF illustrates these steps.
 
 <img src="{{site.url}}{{site.baseurl}}/images/dashboards/notification_mds1.gif" alt="Multiple data sources in Notification plugin"/>
 
 ### Search relevance
 
-When the data source feature is enabled, you can navigate to **Search Relevance** under the **OpenSearch Plugins** menu. On the navigation bar on the upper right, you can view all available data sources. To compare search results between indexes from different data sources, first select a data source and an index for **Query 1**, and then select a data source and an index for **Query 2**. Select **Search** to run the queries. The following GIF illustrates these steps. 
+When you have set `data_source.enabled:true`, you can you can compare search results across indexes from different data sources. 
+
+1. Navigate to **OpenSearch Plugins** > **Search Relevance** under the main menu.
+2. Select the {::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/database-icon.png" class="inline-icon" alt="database icon"/>{:/} icon and choose a data source from the dropdown. A list of available data sources is displayed.
+3. Under **Query 1**, select a data source and an index, and then under **Query 2**, select a data source and an index.
+4. Select the **Search** button to run the queries. The query results are displayed in their respective results panel. 
+
+The following GIF illustrates these steps.
 
 <img src="{{site.url}}{{site.baseurl}}/images/dashboards/searchrelevance_mds1.gif" alt="Multiple data sources in Search Relevance plugin"/>
 
-## Next steps 
+### Security analytics
+Introduced 2.15
+{: .label .label-purple}
 
-After configuring multiple data sources, you can analyze the data from each source. Refer to the following resources for more information:
+When you have set `data_source.enabled:true`, you can view and manage security analytics resources, such as detection rules, across multiple connected data sources.
+
+1. Navigate to **OpenSearch Plugins** > **Security analytics** under the main menu.
+2. Select the {::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/database-icon.png" class="inline-icon" alt="database icon"/>{:/} icon and choose a data source from the dropdown.
+3. Select **Dectectors** > **Detection rules** from the left-side navigation menu. A list of detection rules is displayed.
+4. Select a rule to open the details pop-up window.
```

Comment:
If "details" is the name of the pop-up window, then it should be formatted as such. If it is not, we should say something like "Select a rule to open a pop-up window containing more information about that rule."